src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -143,23 +143,33 @@ namespace winrt::TerminalApp::implementation
         _root = winrt::make_self<TerminalPage>(*_WindowProperties, _manager);
         _dialog = ContentDialog{};
 
-        // Pass commandline args into the TerminalPage. If we were supposed to
-        // load from a persisted layout, do that instead.
-
-        // layout will only ever be non-null if there were >0 tabs persisted in
-        // .TabLayout(). We can re-evaluate that as a part of TODO: GH#12633
-        if (const auto& layout = LoadPersistedLayout())
+        // Pass in information about the initial state of the window.
+        // * If we were suppost to start from serialized "content", do that,

```

Comment:
```suggestion
        // * If we were supposed to start from serialized "content", do that,
```

Reply:
you marked as resolved, but the comment still shows as "suppost" for me?